src/main/java/com/robinpowered/sdk/model/Device.java

```diff
@@ -91,10 +92,18 @@ public DateTime getUpdatedAt() {
         return identifiers;
     }
 
+    public Space getSpace() {
+        return space;
+    }
+
     public void setIdentifiers(List<Identifier> identifiers) {
         this.identifiers = identifiers;
     }
 
+    public void setSpace(Space space) {
+        this.space = space;
+    }
```

Comment:
Ah didn't realize we did
```
getX
setX
getY
setY
getZ
setZ
```

thought we separated getters/setters
```
getX
getY
getZ
setX
setY
setZ
```

👍  will change